src/audio/audio-recorder.js

```diff
@@ -258,20 +261,25 @@ export default class AudioRecorder {
       throw new Error('Already recording, stop recording first.');
     }
 
-    this.audioContext.resume();
+    const delay = disableDelay ? 0 : 100;
```

Comment:
i'd rename parameter to: recordingDelay=100 by default, override to 0 or 250 or whatever you want by sdk user.

Reply:
to be honest, I don't think we should allow to override this value (at this point). Maybe our future recorder could provide also a way to override the value specifically but at this point I made the choice to either allow the default delay (or not).

Reply:
I don't have very strong opinions here. so. fine, i'll merge as is.